### PR TITLE
[no ticket] bumps a few versions

### PR DESCRIPTION
**Pull request by Qi77Qi**
Closed a bunch scala-steward PRs.

Removed a few non required explicit dependencies, and bumped a few.

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by Qi77Qi** [comment]
jenkins retest

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala [hunk 1]
```diff
@@ -817,10 +817,13 @@ object RuntimeServiceInterp {
       config.autoFreezeConfig
     ) //TODO: use FiniteDuration for autopauseThreshold field in Cluster
     val clusterScopes = req.runtimeConfig match {
-      case Some(rq) if rq.cloudService == CloudService.GCE =>
-        if (req.scopes.isEmpty) config.gceConfig.defaultScopes else req.scopes
-      case Some(rq) if rq.cloudService == CloudService.Dataproc =>
-        if (req.scopes.isEmpty) config.dataprocConfig.defaultScopes else req.scopes
+      case Some(rq) =>
```
Comment (rtitle) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala:820]: This is probably just scalafix, but maybe this is a little more concise:
```
    val clusterScopes =
      if (req.scopes.nonEmpty) req.scopes
      else
        req.runtimeConfig.map(_.cloudService) match {
          case Some(CloudService.Dataproc) => config.dataprocConfig.defaultScopes
          case _                           => config.gceConfig.defaultScopes
        }
```

**Review by rtitle — APPROVED** [review]

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala (hunk 1 above)
Comment (Qi77Qi) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala:820]: oops...forgot to fix this before merging...it wasn't scalafix, I didn't really read the code but was just mechanically fixing some syntex error

Comment (rtitle) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala:820]: np, not too important (i still approved the PR)

**Pull request merged**